stdlib/src/bit/bit.mojo

```diff
@@ -222,7 +270,6 @@ fn pop_count[
 # ===----------------------------------------------------------------------===#
 # bit_not
 # ===----------------------------------------------------------------------===#
-# TODO: implement bit_not for Int type
```

Comment:
`bit_not(val)` is the same as `~val` so I do not find the need to wrap that into a function.